website/docs/explorer/getting-started.mdx

```diff
@@ -16,7 +16,7 @@ This guide shows you the basics steps to start using Explorer.
 
 Before using Explorer, please ensure that:
 
-- You have Weave Gitops Enterprise [v0.21.2](../../releases)
+- You have Weave Gitops Enterprise [v0.21.2](../enterprise/getting-started/releases-enterprise.mdx)
```

Comment:
Should we say "at least v0.23.0" or something? (in a follow-up PR)

Reply:
Good catch -- I made a change to reflect this